<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Nedrysoft::Ribbon::RibbonWidget" name="fileRibbonGroup">
      <widget class="QWidget" name="File">
       <attribute name="title">
        <string>File</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="icons">
       <attribute name="title">
        <string>Icons</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="build">
       <attribute name="title">
        <string>Build</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Nedrysoft::PreviewWidget" name="previewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Nedrysoft::Ribbon::RibbonWidget</class>
   <extends>QTabWidget</extends>
   <header>Ribbon/RibbonWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Nedrysoft::PreviewWidget</class>
   <extends>QWidget</extends>
   <header>PreviewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
